tests/local_contract_calls.py

```diff
@@ -91,8 +91,10 @@ class ReceiptAggregateVoucher(EIP712Message):
 except ContractCustomError as e:
     print("Custom Error: %s" % e)
     print(decode_custom_error(erc20_abi_json, str(e), w3))
+    sys.exit(1)
```

Comment:
Or even better, you can just add more context to the error directly. It will also print the original exception. (and it will output on stderr, better than vanilla printing):
```suggestion
except ContractCustomError
    raise Error(decode_custom_error(erc20_abi_json, str(e), w3))
```
Read about "exception chaining" here: https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement